auslib/db.py

```diff
@@ -946,7 +946,22 @@ def getReleaseInfo(self, product=None, limit=None,
             column = [self.name]
         else:
             column = [self.name, self.product, self.data_version, self.read_only]
+
         rows = self.select(where=where, columns=column, limit=limit, transaction=transaction)
+        j = join(dbo.releases.t, dbo.rules.t, dbo.releases.name == dbo.rules.mapping)
+        ref_list = select([dbo.releases.name, dbo.rules.rule_id]).select_from(j).execute().fetchall()
```

Comment:
This query is a good start! As I mentioned on IRC, we need to include rules that point at whitelists as well. Ideally, we'd get them all from one query. If that's not possible, you could do a second query and merge the results.